docs/sources/explore/logs-integration.md

```diff
@@ -0,0 +1,120 @@
++++
+title = "Logs in Explore"
+description = "Logs in Explore"
+keywords = ["explore", "logs",]
+aliases = ["/docs/grafana/latest/features/explore/"]
+weight = 15
++++
+
+# Logs in Explore 
+
+Along with metrics, Explore allows you to investigate your logs in the following data sources:
+
+- [Loki](../datasources/loki)
+- [InfluxDB](../datasources/influxdb)
+- [Elasticsearch](../datasources/elasticsearch)
+
+During an infrastructure monitoring and incident response, you can dig deeper into the metrics and logs to find the cause. Explore also allows you to correlate metrics and logs by viewing them side-by-side. This creates a new debugging workflow where you can:
+
+1. Receive an alert
+2. Drill down and examine metrics
```

Comment:
Conceptually, "drill down" and "examine" mean the same thing. "Drill down" is much less commonly used than "examine", so I recommend using "examine". Is there something a user is doing to allow them to examine metrics more closely? How does a user drill down in this context?

Can we collapse all three terms that mean the same thing: "dig deeper", "drill down", and "examine"?

Reply:
We can revisit this, but I think all three terms have slightly different meanings, and the content is good for now.